core/dbt/contracts/graph/nodes.py

```diff
@@ -1579,13 +1579,12 @@ def same_exports(self, old: "SavedQuery") -> bool:
 
         # exports should be in the same order, so we zip them for easy iteration
         for old_export, new_export in zip(old.exports, self.exports):
-            if not (
-                old_export.name == new_export.name
-                and old_export.config.export_as == new_export.config.export_as
-                and old_export.config.schema_name == new_export.config.schema_name
-                and old_export.config.alias == new_export.config.alias
-            ):
+            if not (old_export.name == new_export.name):
```

Comment:
The check is out of date, the newest one show 100% coverage https://github.com/dbt-labs/dbt-core/pull/10565/checks?check_run_id=29180771199